### PR TITLE
NEW_FILE

### PR DESCRIPTION
**Pull request by dadakiren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,14 +28,7 @@ Try it! It’s fun, we promise :)
 
 By the way, creating issues is fun too!
 
----
```
Comment (suchov) [inline README.md:31]: What "Andrii" did to you, so you decided to remove his lines?

File: README.md [hunk 2]
```diff
@@ -28,14 +28,7 @@ Try it! It’s fun, we promise :)
...
-## Hi kottans, my name is Andrii.
-I'm late, but I hope, I'll can study with you!
-All the best!
-___
-## Hi everyone!!!
 
-My name Aleksandr and i start my way in Frontend development (i have little expirience).
```
Comment (suchov) [inline README.md:38]: And waht about "Aleksandr"?

File: README.md [hunk 3]
```diff
@@ -4064,3 +4057,7 @@ My name is [Kostiantyn](https://github.com/rukhlovka)! And this is my cat and my
 
 ## Hello, Kottans
 My name is Alex.
+
```
Comment (suchov) [inline README.md:4060]: And this looks good.

**Review by suchov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dadakiren) [reply README.md]: I removed lines "Andrii"

File: README.md (hunk 2 above)
Comment (dadakiren) [reply README.md]: I removed lines "Aleksandr". Thank's.

**Review by suchov — APPROVED** [review]

**Pull request merged**